### PR TITLE
v3 - consultants in project

### PR DESCRIPTION
**Pull request by mathiazom**
See #815 

Shows the employee cards for each employee in the project consultants list. Data fetched from Chewbacca.

The project consultants field should now contain the employee emails.

image

#### Desktop

image

#### Mobile
image

### CODE REVIEW

File: studioShared/schemas/fields/customerCaseProjectInfo.ts [hunk 1]
```diff
@@ -36,9 +36,10 @@ export const customerCaseProjectInfo = defineField({
     defineField({
       // TODO: We should be able to select the consultants from a list
       name: "consultants",
-      description: "The consultants for the project",
+      description:
```
Comment (anemne) [inline studioShared/schemas/fields/customerCaseProjectInfo.ts:39]: Do we need to use @variant.no or could we only use "amn"?

File: src/components/customerCases/customerCase/CustomerCase.tsx [hunk 2]
```diff
@@ -54,12 +63,14 @@ export default function CustomerCase({ customerCase }: CustomerCaseProps) {
                 {customerCase.projectInfo.delivery}
               </Text>
             </div>
-            <div className={styles.projectInfoItem}>
-              <Text>Konsulenter</Text>
-              <Text className={styles.projectInfoItemValue}>
-                {customerCase.projectInfo.consultants.join(", ")}
-              </Text>
-            </div>
+            {consultantsResult.ok && (
```
Comment (anemne) [inline src/components/customerCases/customerCase/CustomerCase.tsx:66]: isn't it enough to write consultantsResult?

File: studioShared/schemas/fields/customerCaseProjectInfo.ts (hunk 1 above)
Comment (mathiazom) [reply studioShared/schemas/fields/customerCaseProjectInfo.ts:39]: We are not sure enough that they are unique across countries (e.g. mom@variant.no and mom@variant.se could be two different people). So going for the safer option now. The user experience may be a little worse, but that could be fixed with a dropdown/search solution (where the internal value does not matter to the user)

File: src/components/customerCases/customerCase/CustomerCase.tsx [hunk 3]
```diff
@@ -91,6 +102,50 @@ export default function CustomerCase({ customerCase }: CustomerCaseProps) {
...
+                          alt={consultant.name}
+                          objectFit="cover"
+                          fill={true}
+                        />
+                      </div>
+                      <div className={styles.consultantInfo}>
+                        <p className={styles.consultantName}>
+                          {consultant.name}
+                        </p>
+                        {consultant.officeName && (
```
Comment (anemne) [inline src/components/customerCases/customerCase/CustomerCase.tsx:127]: I think it's nice to have office location, but we should probably ask designers

File: src/components/customerCases/customerCase/CustomerCase.tsx (hunk 2 above)
Comment (mathiazom) [reply src/components/customerCases/customerCase/CustomerCase.tsx:66]: Are you thinking just `consultantsResult && ...`?

This is using the `Result` type, so `consultantsResult` will always be an object. So checking if `consultantsResult` is truthy would always return true. The proper check with `Result` is to check if `.ok` is true.

**Review by anemne — APPROVED** [review]
I think this looks good!

**Pull request merged**